### PR TITLE
Magnite Analytics: Minor fix for default rule name

### PR DESCRIPTION
**Pull request by robertrmartinez**
## Type of change
- [X] Bugfix

## Description of change
We would not send `rule` if did not detect cookieless which is not right! Set default now.

### CODE REVIEW

File: test/spec/modules/magniteAnalyticsAdapter_spec.js [hunk 1]
```diff
@@ -1758,16 +1758,39 @@ describe('magnite analytics adapter', function () {
...
-        magniteAdapter.enableAnalytics({
-          options: {
-            cookieles: undefined
-          }
-        });
-      })
+      // beforeEach(() => {
+      //   magniteAdapter.enableAnalytics({
+      //     options: {
+      //       cookieles: undefined
```
Comment (patmmccann) [inline test/spec/modules/magniteAnalyticsAdapter_spec.js:1764]: it is on the comment now, but was spelling causing an issue?

File: test/spec/modules/magniteAnalyticsAdapter_spec.js [hunk 2]
```diff
@@ -1758,16 +1758,39 @@ describe('magnite analytics adapter', function () {
...
+      //     }
+      //   });
+      // })
       afterEach(() => {
         magniteAdapter.disableAnalytics();
       })
+      it('should not add cokieless and preserve original rule name', () => {
```
Comment (patmmccann) [inline test/spec/modules/magniteAnalyticsAdapter_spec.js:1771]: cookieless

File: test/spec/modules/magniteAnalyticsAdapter_spec.js (hunk 1 above)
Comment (robertrmartinez) [reply test/spec/modules/magniteAnalyticsAdapter_spec.js]: no it was something else (mot initializing the variable), this bit of code just does not even do anything so I commented out while testing and forgot to remove, I deleted now.

File: test/spec/modules/magniteAnalyticsAdapter_spec.js (hunk 2 above)
Comment (robertrmartinez) [reply test/spec/modules/magniteAnalyticsAdapter_spec.js]: fixed!

**Review by patmmccann — APPROVED** [review]

**Pull request merged**